jython/Jynstruments/ThrottleFrame/VideoViewIP.jyn/VideoViewIP.py

```diff
@@ -16,11 +28,17 @@
 import java.awt.BorderLayout as BorderLayout
 import java.awt.event.ItemListener as ItemListener
 import javax.swing.JCheckBoxMenuItem as JCheckBoxMenuItem
-import com.github.sarxos.webcam.Webcam as Webcam
-import com.github.sarxos.webcam.WebcamPanel as WebcamPanel
-import com.github.sarxos.webcam.ds.ipcam.IpCamDeviceRegistry as IpCamDeviceRegistry
-import com.github.sarxos.webcam.ds.ipcam.IpCamDriver as IpCamDriver
-import com.github.sarxos.webcam.ds.ipcam.IpCamMode as IpCamMode
+import javax.swing.JOptionPane as JOptionPane
+import javax.swing.JTextArea as JTextArea
+import javax.swing.JFrame as JFrame
+try:
+	import com.github.sarxos.webcam.Webcam as Webcam
+	import com.github.sarxos.webcam.WebcamPanel as WebcamPanel
+	import com.github.sarxos.webcam.ds.ipcam.IpCamDeviceRegistry as IpCamDeviceRegistry
+	import com.github.sarxos.webcam.ds.ipcam.IpCamDriver as IpCamDriver
+	import com.github.sarxos.webcam.ds.ipcam.IpCamMode as IpCamMode
```

Comment:
These lines starts with a TAB character. It would be good if you could replace the TAB character with spaces. The problem with TAB is that different editors have different settings for how many spaces on TAB character represent.

Reply:
done.